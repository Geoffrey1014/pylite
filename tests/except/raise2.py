class X:
    pass

def main():
    raise X

print "Start"
main()
print "End"
