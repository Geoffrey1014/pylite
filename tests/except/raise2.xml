lambda
  name __init__
  param_count 0
  local_count 0
  label_count 0
  insn_vector
    new_class c0
    store_global c0
    load_const c1
    store_global c2
    load_const c3
    print
    load_global c2
    call 0
    pop
    load_const c4
    print
    load_const c5
    return
  const_vector
    X
    lambda
      name main
      param_count 0
      local_count 0
      label_count 0
      insn_vector
        load_global c0
        raise
        load_const c1
        return
      const_vector
        X
        None
      guard_vector
      line_number_map 5 5 4 4
    main
    "Start"
    "End"
    None
  guard_vector
  line_number_map 1 1 4 4 7 7 8 8 8 9 9 1 1
