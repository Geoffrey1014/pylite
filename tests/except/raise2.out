Start
TypeError:
  At raise2.py, line 5, in main()
  At raise2.py, line 8, in __init__()
